src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,17 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    enum Color {
+        RED,
+        GREEN,
+        BLUE
+    }
+
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
Overloaded expression, please introduce some variables. Also creation of Random object should be in class level variable